cas/ctg_store.c

```diff
@@ -966,6 +966,10 @@ static void ctg_store_release(struct m0_ref *ref)
 
 	M0_ENTRY();
 	m0_mutex_fini(&ctg_store->cs_state_mutex);
+	// TODO: Cleanup all in-memory allocations done for indices in meta tree
```

Comment:
Also if TODO comment can be updated "Clean up every index in memory tree allocation upon any CAS operation on the index " ?

Reply:
Updated.